code/modules/client/preference_setup/loadout/lists/fluff.dm

```diff
@@ -0,0 +1,192 @@
+	// ckeys are always lowercase
+	var/list/ckey_whitelist = list()
+
+	// Typed like this to hint at what typepath it should contain
+	// as well as used to avoid a recast in examine()
+	var/obj/target_type = null
+
+	// Multiple options: Spawning a new item
+	var/path_change = null
+
+	// Or modifying the existing items vars
+	var/name_change = null
+	var/icon_change = null
+	var/icon_state_change = null
+	var/vars_change = list()
+
+/obj/item/fluff_conversion_kit/examine(user, distance)
+	..()
+	to_chat(user, SPAN_NOTICE("This is able to convert '[initial(target_type.name)]'."))
+
+/obj/item/fluff_conversion_kit/afterattack(atom/target, mob/user, proximity_flag, params)
+	if(!proximity_flag)
+		return
+	if(!istype(user))
+		return
+	if(user.incapacitated())
+		return
+	if(!user.IsAdvancedToolUser())
+		return
+
+	if(!target_type)
+		CRASH("Fluff conversion kit [name] doesn't specify a target type!")
+
+	if(!istype(target, target_type))
+		to_chat(user, "[src] does not support '[target]'.")
+		return
+
+	user.visible_message("[user] starts to apply [src] to [target]...", "You start to apply [src] to [target]...")
+	if(!do_after(user, 2 SECONDS, target))
+		to_chat(user, "You need to stay still to apply [src].")
+		return
+
+
+	var/old_name = target.name
+	var/old_type = target.type
+	var/obj/final_item = convert(target)
+	user.visible_message("[user] uses [src] to transform '[old_name]' into '[final_item]'.", "You apply [src] to transform '[old_name]' into '[final_item]'.")
+	// Put down a note that this happened
+	final_item.investigate_log("was converted from [old_name] ([old_type]) into a fluff item by [user.ckey] using [name] ([type]).", "fluff")
+	if(!(user.ckey in ckey_whitelist))
+		// And an extra log if it was used suspiciously
+		log_admin("[key_name(user)] used the fluff item modkit '[name]' that is only supposed to be used by ckeys '[jointext(ckey_whitelist, ", ")]'.")
+	qdel(src)
+
+/obj/item/fluff_conversion_kit/proc/convert(atom/target)
+	if(path_change)
+		var/obj/item/new_item = new path_change(target.loc)
+		qdel(target)
+		return new_item
+
+	if(name_change)
+		target.name = name_change
+
+	if(icon_change)
+		target.icon = icon_change
+
+	if(icon_state_change)
+		target.icon_state = icon_state_change
+
+	if(length(vars_change))
+		for(var/key in vars_change)
+			target.vars["[key]"] = vars_change[key]
+
+	target.update_icon()
+	return target
+
+/datum/gear/fluff
+	category = /datum/gear/fluff
+	sort_category = "Fluff Items"
+	cost = 1
+
+/datum/gear/fluff/New()
+	. = ..()
+	// no gear tweaks, they won't be carried through correctly because of modkits
+	gear_tweaks = list()
+
+/datum/gear/fluff/spawn_item(location, metadata)
+	. = ..()
+	var/obj/item/fluff_conversion_kit/kit = .
+	if(istype(kit))
+		kit.ckey_whitelist = ckey_whitelist
+
+
+/**********************************/
+/*                                */
+/*    PLAYER STUFF BEYOND HERE    */
+/*                                */
+/**********************************/
+
```

Comment:
Being a uniquely server-facing feature, the actual list of fluff items, and their icons, should be moved into its own unique directory under the project root.

Reply:
(By server-facing, I mean other derivative servers should be able to safely replace this file with their own copy and *ignore* upstream changes. There are git commands that do this. The fluff items of our playerbase are not relevant to downstreams.)